internal/dao/pod.go

```diff
@@ -491,3 +499,20 @@ func (p *Pod) isControlled(path string) (string, bool, error) {
 	}
 	return "", false, nil
 }
+
+func getDefaultLogContainer(po v1.Pod) string {
+	defaultContainer := po.GetAnnotations()[defaultLogContainerAnnotation]
+	if defaultContainer == "" {
+		return ""
+	}
+	for _, container := range po.Spec.Containers {
+		if container.Name == defaultContainer {
+			return defaultContainer
+		}
+	}
+	if defaultContainer != "" {
+		log.Info().Msg(defaultLogContainerAnnotation + " annotation is set to " + defaultContainer + ", but there is no container with this name. Ignoring kubectl.kubernetes.io/default-logs-container annotation")
```

Comment:
If is superfluous! Just return an "", error here. I think best to have getDefaultContainer return string, error.

Reply:
done